src/panels/developer-tools/statistics/developer-tools-statistics.ts

```diff
@@ -43,6 +45,8 @@ class HaPanelDevStatistics extends LitElement {
     state?: HassEntity;
   })[] = [] as StatisticsMetaData[];
 
+  @state() private _disabledEntities = new Set<string>();
```

Comment:
`_disabledEntities` is not used in render/updated, so it doesnt need to be a `state`